osu.Game/Audio/BassAudioNormalization.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using ManagedBass;
+using ManagedBass.Loud;
+
+namespace osu.Game.Audio
+{
+    public class BassAudioNormalization
+    {
+        public float IntegratedLoudness { get; }
+
+        public BassAudioNormalization(string filePath)
+        {
+            int decodeStream = Bass.CreateStream(filePath, 0, 0, BassFlags.Decode | BassFlags.Float);
```

Comment:
Will probably have to move `BassAudioNormalization` to o!f to do this. Also I'm not entirely certain on how I would a data stream to pass to `FileCallbacks`.

I've mostly been avoiding o!f since I haven't touched its code or read it at all, as opposed to osu! where I've spent a good while going through the codebase and looking at other pr's to get familiar with it. It'll take me a while to get familiar with o!f to the point where I'll be able to figure out how to pass the correct information here.

I'll probably address this after I finish moving things to and from o!f